.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
same, I think this is carry over from a different PR? https://github.com/facebookresearch/torchsnapshot/pull/17/